docs/CONFIG-PROPERTIES.md

```diff
@@ -23,7 +23,7 @@
 | timer.port.timeout | timer in seconds | 15 | time for each http/send |
 | timer.port.testbetterinterval | timer in seconds | 600 | test a higher prio port config |
 | network.fallback.any.eth | "enabled" or "disabled" | enabled | if no connectivity try any Ethernet, WiFi, or LTE |
-| network.allow.wwan.download | "enabled" or "disabled" | disabled | allow image download over non-free ports like LTE |
+| network.download.priority | 0-255 | 0 | allow image download over lower-priority ports like LTE |
```

Comment:
Boolean was self-documenting, but I don't think int is -- IOW, what happens if this value is 42?

Reply:
Fixed.